### PR TITLE
feat(messaging): added go doc

### PR DESCRIPTION
**Pull request by RoRoJ**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Add Go doc to API/CLI section

### CODE REVIEW

File: compute/messaging/api-cli/go-sns.mdx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  Name: aws.String("my-test-topic"),
+})
+fmt.Println(*createTopicResponse.TopicArn)
+```
+
+## Publish messages to this topic
+
+Be careful: messages sent to topics with no subscriptions are automatically deleted
+
+```goYour location
```
Comment (albinou) [inline compute/messaging/api-cli/go-sns.mdx:93]:
```suggestion
```go
```

File: compute/messaging/api-cli/go-sqs.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+  awsSqs := sqs.New(awsSession)
+
+  [...]
+}
+```
+
+<Message type="note">
+
+  The `endpoint_url` for Scaleway Messaging is `http://sqs-sns.mnq.fr-par.scw.cloud`. For the access and secret key values, use the credentials you [generated](/compute/messaging/how-to/create-credentials/) for your SNS/SQS namespace.
```
Comment (albinou) [inline compute/messaging/api-cli/go-sqs.mdx:74]:
```suggestion
  The `Endpoint` for Scaleway Messaging is `http://sqs-sns.mnq.fr-par.scw.cloud`. For the access and secret key values, use the credentials you [generated](/compute/messaging/how-to/create-credentials/) for your SNS/SQS namespace.
```

**Review by albinou — CHANGES_REQUESTED** [review]
Looks good. Just a few small comments.

**Comment by albinou** [comment]
I can't approve, but it looks great now :+1:

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**